index.html

```diff
@@ -9,42 +9,57 @@
-            <li class="letter hidden" id="letter-2">_</li>
-            <li class="letter hidden" id="letter-3">_</li>
-            <li class="letter hidden" id="letter-4">_</li>
-            <li class="letter hidden" id="letter-5">_</li>
-            <li class="letter hidden" id="letter-6">_</li>
-            <li class="letter hidden" id="letter-7">_</li>
-            <li class="letter hidden" id="letter-8">_</li>
-            <li class="letter hidden" id="letter-9">_</li>
-        </ul>
-    </section>
-
-    <section id="stage">
-        This is where the stage will go to show player progress
-    </section>
+    <header>
+        <h1>Guess or Die Game!</h1>
+    </header>
+    <main>
+        
+        <section id="output">
+            <ul class="bananas">
+                <li class="letter hidden" id="letter-0">_</li>
+                <li class="letter hidden" id="letter-1">_</li>
+                <li class="letter hidden" id="letter-2">_</li>
+                <li class="letter hidden" id="letter-3">_</li>
+                <li class="letter hidden" id="letter-4">_</li>
+                <li class="letter hidden" id="letter-5">_</li>
+                <li class="letter hidden" id="letter-6">_</li>
+                <li class="letter hidden" id="letter-7">_</li>
+                <li class="letter hidden" id="letter-8">_</li>
+                <li class="letter hidden" id="letter-9">_</li>
+            </ul>
+        </section>
+        
+        <section>
+                <h3>Guessed Letters</h3>
+                <p id = "letters"></p>
+        </section>
+        
+        <section id="guess">
+        <form id="answer-field" onsubmit="event.preventDefault(); onSubmit();"  onreset="resetButton()" >
+                <input type="text" maxlength="1" name="guess"  id="guess" placeholder="enter a letter"/>
+                <button type="submit" id="mySubmit">SUBMIT</button>
+                <button type="reset">RESET</button>
+                </form>
+        </section>
+        
+        <section id="stage">
+            <ul class="help">
+                <li id="gallows-0" class="gallow hidden"><img src="./assets/hangman_00.png" alt="hangman"></div>
+                <li id="gallows-1" class="gallow hidden"><img src="./assets/hangman_01.png" alt="hangman"></div>
+                <li id="gallows-2" class="gallow hidden"><img src="./assets/hangman_02.png" alt="hangman"></div>
+                <li id="gallows-3" class="gallow hidden"><img src="./assets/hangman_03.png" alt="hangman"></div>
+                <li id="gallows-4" class="gallow hidden"><img src="./assets/hangman_04.png" alt="hangman"></div>
+                <li id="gallows-5" class="gallow hidden"><img src="./assets/hangman_05.png" alt="hangman"></div>
+                <li id="gallows-6" class="gallow hidden"><img src="./assets/hangman_06.png" alt="hangman"></div>
+            </ul>
+        </section>
 
-    <aside id="heads-up-display">
         <section id="turns-total">
```

Comment:
Looks like you didn't get to this section, but it does count as dead code because it is unused. It's always good to leave a ```//TODO:``` comment for code like this so it's clear you meant to come back.